src/components/reportPage/ReportPage.jsx

```diff
@@ -24,6 +27,33 @@ const ReportPage = (props) => {
   const [colourScheme, setColourScheme] = useState(inferno256);
   const [searchTerm, setSearchString] = useState("");
 
+  const [filterType, setFilterType] = useState("All");
+  const [videoFilterType, setVideoFilterType] = useState("0");
+
+  const [displayValue, getValue] = useState([]);
```

Comment:
Could we think of a better name for this? It's not very clear what this list is. Also should be `set` not `get`.